src/Microsoft.AspNet.Razor/Generator/Compiler/CodeBuilder/CSharp/Visitors/CSharpCodeVisitor.cs

```diff
@@ -424,25 +453,30 @@ private void RenderRuntimeExpressionBlockChunkWithContentSpan(ExpressionBlockChu
             {
                 if (!string.IsNullOrEmpty(Context.TargetWriterName))
                 {
-                    var generatedStart = Context.Host.GeneratedClassContext.WriteToMethodName.Length +
-                                         Context.TargetWriterName.Length +
-                                         3; // 1 for the opening '(' and 2 for ', '
+                    var generatedStart =
+                        WriteToMethodName.Length +
+                        Context.TargetWriterName.Length +
+                        3; // 1 for the opening '(' and 2 for ', '
 
                     var padding = _paddingBuilder.BuildExpressionPadding(contentSpan, generatedStart);
 
-                    Writer.Write(padding)
-                          .WriteStartMethodInvocation(Context.Host.GeneratedClassContext.WriteToMethodName)
-                          .Write(Context.TargetWriterName)
-                          .WriteParameterSeparator();
+                    Writer
+                        .Write(padding)
+                        .WriteStartMethodInvocation(WriteToMethodName)
+                        .Write(Context.TargetWriterName)
+                        .WriteParameterSeparator();
                 }
                 else
                 {
-                    var generatedStart = Context.Host.GeneratedClassContext.WriteMethodName.Length +
-                                         1; // for the opening '('
+                    var generatedStart =
+                        WriteMethodName.Length +
+                         1; // for the opening '('
```

Comment:
nah.  as-is the comment has a clear referent and is easily compared with line 459